modules/sandboxed-containers-deleting-namespace-web-console.adoc

```diff
@@ -0,0 +1,24 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-namespace-web-console_{context}"]
+= Deleting a namespace using the web console
+
+You can delete a namespace by using the {product-title} web console.
+[NOTE]
```

Comment:
My $0.02 is that this is better in the midst of the procedure and probably just as a sentence.

```
. On the rightmost side...
+
If the *Delete Namespace* option is not available, you do not have permission to delete the namespace.
```
My sense is that there isn't a useful way to determine the permission before beginning.  Break the bad news at the last minute.

If you prefer to keep this as a NOTE and here, then I'd add a blank line before it.